libs/langgraph/src/pregel/loop.ts

```diff
@@ -71,14 +74,13 @@ import { LangGraphRunnableConfig } from "./runnable_types.js";
 const INPUT_DONE = Symbol.for("INPUT_DONE");
 const INPUT_RESUMING = Symbol.for("INPUT_RESUMING");
 const DEFAULT_LOOP_LIMIT = 25;
-const SPECIAL_CHANNELS = [ERROR, INTERRUPT];
 
 // [namespace, streamMode, payload]
 export type StreamChunk = [string[], StreamMode, unknown];
 
 export type PregelLoopInitializeParams = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  input?: any;
+  input?: any | Command;
```

Comment:
Will this even give devs autocomplete since `any` is allowed here? If not, I think we should drop `Command` since `any` will cover it

Reply:
oh wait this isnt externally facing. I think just drop then